base/message/example/verifiable-presentation.json

```diff
@@ -0,0 +1,12 @@
+{
+    "id": "https://example.com/credentials/1872",
+    "type": ["VerifiablePresentation"],
```

Comment:
Do we need the `"VerifiablePresentation"` object at all? Is it referenced somewhere? I couldn't find anything in the spec.

Reply:
Yes, it is required in PresentationResponseMessage schema